packages/scripts/src/upgrade-tests/voting2/1_Propose.ts

```diff
@@ -204,23 +215,37 @@ async function main() {
     const iface = new hre.ethers.utils.Interface(getAbi("MultiRole"));
     const data = iface.encodeFunctionData("resetMember", [0, votingUpgrader.address]);
     adminProposalTransactions.push({ to: contractAddress, value: 0, data });
-    console.log(`2.i.  Multirole: transfer owner role of ${contractName} to voting upgrader`, data);
+    console.log(`2.j.  Multirole: transfer owner role of ${contractName} to voting upgrader`, data);
+  }
+
+  const resetProposerGovernorTx = await governor.populateTransaction.resetMember(1, governorV2.address);
+  if (!resetProposerGovernorTx.data) throw "resetProposerGovernorTx.data is null";
+  adminProposalTransactions.push({ to: governor.address, value: 0, data: resetProposerGovernorTx.data });
+  console.log("2.k.  Reset governor proposer to governor v2:", resetProposerGovernorTx.data);
+
+  if ((await governorV2.getMember(1)) != proposerV2.address) {
+    if (!process.env[TEST_DOWNGRADE]) throw "ProposerV2 is not registered in GovernorV2";
```

Comment:
This only runs when downgrading. When upgrading this role setting is done [here](https://github.com/UMAprotocol/protocol/blob/md0x/votingv2-upgrade-updates/packages/scripts/src/upgrade-tests/voting2/0_Deploy.ts#L188)